packages/vitest/package.json

```diff
@@ -85,6 +85,7 @@
   "dependencies": {
     "@types/chai": "^4.3.0",
     "@types/chai-subset": "^1.3.3",
+    "benchmark": "^2.1.4",
```

Comment:
I haven't looked close to it but seems this package is no longer maintained for 5 years already. Do we have a better alternative?

Reply:
I saw the issues recommendation and didn't look for other 🙈

Reply:
@antfu There seems to be no better alternative https://www.npmjs.com/search?q=keywords:benchmark. I think it is possible to fork `benchmark` to maintain a new one?

Reply:
There is a fork by @Uzlopak actively updated in https://github.com/cthulhu-node/benchmark.js might be worthy joining forces

Reply:
I am now back from a three weeks vacation. I still want to implement the ability to automatically detect async functions and add the one PR to the original project which was fixing the statistics.
I did not publish a package yet. 
But we can surely join forces. :)